packages/expect/src/__tests__/extend.test.ts

```diff
@@ -69,6 +64,17 @@ declare module '../types' {
   }
 }
 
+const jestExpect = importedExpect as typeof importedExpect & {
```

Comment:
In this case typings have to be added like this. TypeScript is not happy about merging declaration of `AsymmetricMatchers` interface – it goes looking for `expect` object in `../index.ts`, does not find properties from the declaration and complains.